WindowsFunctionsStacks.json

```diff
@@ -133,16 +161,18 @@
                         "displayVersion": "11",
                         "runtimeVersion": "11",
                         "supportedFunctionsExtensionVersions": [
+                            "~2",
                             "~3"
                         ],
-                        "isDefault": true,
+                        "isDefault": false,
```

Comment:
@amamounelsayed  
- Please confirm default java version on V2 and default java version on V3 for windows
- Please confirm default java version on V2 and default java version on V3 for linux

Reply:
Please confirm default java version on V2 and default java version on V3 for windows --> Java8
Please confirm default java version on V2 and default java version on V3 for linux --> java8

Reply:
Do you mind explaining why Java 8 is default for Windows V3 still? If it is no longer preview we should default to the latest version.

Reply:
My understanding should we have java 11 as preview? 

cc @anirudahgarg